metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_lineage_v2.py

```diff
@@ -413,8 +413,9 @@ def _process_upstream_lineage_row(
             return UpstreamLineageEdge.parse_obj(db_row)
         except Exception as e:
             self.report.num_upstream_lineage_edge_parsing_failed += 1
+            upstream_tables = db_row.get("UPSTREAM_TABLES")
             self.structured_reporter.warning(
-                "Failed to parse lineage edge",
+                f"Failed to parse lineage edge for upstream tables {upstream_tables}",
                 context=db_row.get("DOWNSTREAM_TABLE_NAME") or None,
```

Comment:
```suggestion
                "Failed to parse lineage edge",
                context=f"Upstreams: {upstream_tables} Downstreams: {db_row.get('DOWNSTREAM_TABLE_NAME') or None}",
```

first arg `message` is recommended to be free of variables for efficient grouping on UI across errors. Arg `context` can contain such dynamic variables.